src/quick-lint-js/parse.h

```diff
@@ -741,9 +741,18 @@ class parser {
     }
     case expression_kind::_typeof: {
       expression *child = ast->child_0();
-      if (child->kind() == expression_kind::variable) {
+      switch (child->kind()) {
+      case expression_kind::conditional: {
+        if (child->child_0()->kind() == expression_kind::variable) {
```

Comment:
Blocking: This causes false negatives for other cases, like `"undefined" === typeof foo.x ? 3 : 4;`.

Reply:
> We'd want the test cases regardless of what the solution is, no?

Yes